test/e2e/clusterctl_upgrade.go

```diff
@@ -219,6 +223,11 @@ func ClusterctlUpgradeSpec(ctx context.Context, inputGetter func() ClusterctlUpg
 
 		By("THE MANAGEMENT CLUSTER WITH OLDER VERSION OF PROVIDERS WORKS!")
 
+		if input.PreUpgrade != nil {
+			By("Running Pre-upgrade steps against the amnagement cluster")
```

Comment:
```suggestion
			By("Running Pre-upgrade steps against the management cluster")
```
nit, typo (same below)